### ISSUE TITLE
Find alternative to Actioner which will be shutting down on Aug 31st 2024 

### ISSUE THREAD

**Issue by aherzberg** [department-of-veterans-affairs/va-mobile-app#9270]
We use Actioner to post in slack when a new person is on call and to post maintenance windows in the mobile alerts channel. A viable alternative is using vets-api already built in slack messenger (modules/appeals_api/app/services/appeals_api/slack/messager.rb). or if that doesn't work, Zapier.

**Comment by aherzberg** [comment]
I've investigated using the vets-api slack integration and it'd be possible but will require getting our own api_key and updating devops and manifests repos with the channel_id and api key which is very doable but definitely makes this a bigger effort. I've also talked with Jeff Keene who was an admin on Actioner, and it sounds like the platform may get together and find a new solution to replace all the actioner logic, so I think it'd be a good idea to wait for that to happen so we don't reinvent the wheel and can be consistent with the rest of platform. That will most likely go past the shut down date of actioner but the logic we were using was all nice to have but not pivotal for Mobiles functionality. I'm going to put this in blocked for know unless I've had a chance to Sync with Jeff and other engineers involved with actioner to see what direction they want to go.
